samples/v1.0/Scenarios/Restaurant.json

```diff
@@ -23,13 +23,13 @@
 						},
 						{
 							"type": "TextBlock",
-							"text": "4.5 ★★★★★ (176) · $$",
+							"text": "4.5 stars (176 reviews) · mid-priced",
```

Comment:
It would be nice to investigate if we can extend the extensibility to be able to deal with this kind of scenarios
https://stackoverflow.com/questions/55966205/star-rating-component-what-are-the-correct-accessibility-and-semantics-to-be-u